src/DotNet.Status.Web.Tests/AnnotationsControllerTests.cs

```diff
@@ -0,0 +1,104 @@
+using DotNet.Status.Web.Controllers;
+using DotNet.Status.Web.Options;
+using FluentAssertions;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.DotNet.Web.Authentication.Tests;
+using Microsoft.Extensions.DependencyInjection;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DotNet.Status.Web.Tests
+{
+    [TestFixture]
+    public class AnnotationsControllerTests
+    {
+        private HttpClient _client;
+
+        [SetUp]
```

Comment:
Stateful SetUp methods can lead to weird interactions between tests that you might not expect.  Could you use the "TestData" approach that other tests use?

Reply:
Certainly!